src/@next/components/atoms/Tile/test.tsx

```diff
@@ -54,4 +54,38 @@ describe("<Tile />", () => {
       }
     );
   });
+
+  it("changes style for addNew tile type", () => {
+    const wrapper = mount(
+      <Tile>
+        <p>This is content</p>
+      </Tile>
+    );
+
+    const wrapperWithAddNew = mount(
+      <Tile tileType="addNew">
+        <p>This is content</p>
+      </Tile>
+    );
+
+    expect(wrapper).toHaveStyleRule(
+      "background-color",
+      defaultTheme.tile.backgroundColor
+    );
+
+    expect(wrapperWithAddNew).toHaveStyleRule(
+      "background-color",
+      defaultTheme.colors.secondary,
+      {
+        modifier: ":hover",
+      }
+    );
+    expect(wrapperWithAddNew).toHaveStyleRule(
+      "color",
+      defaultTheme.colors.white,
+      {
+        modifier: ":hover",
+      }
+    );
+  });
 });
```

Comment:
maybe you should add tests for conditionally rendered header and footer?

Reply:
I have added tests for conditional rendering